mis_builder/tests/test_aep.py

```diff
@@ -13,67 +13,81 @@
+        cls.prev_year = cls.curr_year - 1
         # create company
-        self.company = self.res_company.create({"name": "AEP Company"})
+        cls.company = cls.res_company.create({"name": "AEP Company"})
         # create receivable bs account
-        type_ar = self.browse_ref("account.data_account_type_receivable")
-        self.account_ar = self.account_model.create(
+        type_ar = cls.env.ref("account.data_account_type_receivable")
+        cls.account_ar = cls.account_model.create(
             {
-                "company_id": self.company.id,
+                "company_id": cls.company.id,
                 "code": "400AR",
                 "name": "Receivable",
                 "user_type_id": type_ar.id,
                 "reconcile": True,
             }
         )
         # create income pl account
-        type_in = self.browse_ref("account.data_account_type_revenue")
-        self.account_in = self.account_model.create(
+        type_in = cls.env.ref("account.data_account_type_revenue")
+        cls.account_in = cls.account_model.create(
             {
-                "company_id": self.company.id,
+                "company_id": cls.company.id,
                 "code": "700IN",
                 "name": "Income",
                 "user_type_id": type_in.id,
             }
         )
         # create journal
-        self.journal = self.journal_model.create(
+        cls.journal = cls.journal_model.create(
             {
-                "company_id": self.company.id,
+                "company_id": cls.company.id,
                 "name": "Sale journal",
                 "code": "VEN",
                 "type": "sale",
             }
         )
         # create move in December last year
-        self._create_move(
-            date=datetime.date(self.prev_year, 12, 1),
+        cls._create_move(
+            date=datetime.date(cls.prev_year, 12, 1),
             amount=100,
-            debit_acc=self.account_ar,
-            credit_acc=self.account_in,
+            debit_acc=cls.account_ar,
+            credit_acc=cls.account_in,
         )
         # create move in January this year
-        self._create_move(
-            date=datetime.date(self.curr_year, 1, 1),
+        cls._create_move(
+            date=datetime.date(cls.curr_year, 1, 1),
             amount=300,
-            debit_acc=self.account_ar,
-            credit_acc=self.account_in,
+            debit_acc=cls.account_ar,
+            credit_acc=cls.account_in,
         )
         # create move in March this year
-        self._create_move(
-            date=datetime.date(self.curr_year, 3, 1),
+        cls._create_move(
+            date=datetime.date(cls.curr_year, 3, 1),
             amount=500,
-            debit_acc=self.account_ar,
-            credit_acc=self.account_in,
+            debit_acc=cls.account_ar,
+            credit_acc=cls.account_in,
         )
+
+    def setUp(self):
+        super().setUp()
```

Comment:
As we have talked, put the docstring explaining WHY we do this. If not, the reasons are lost with the wind and coming again to this will make us to repeat the same mistakes.

Reply:
Done, thanks for the review.